.github/workflows/deploy_mongo_to_pg.yml

```
@@ -0,0 +1,51 @@
+name: Deploy
+
+run-name: Deploy ${{ inputs.gitRef || github.event.release.tag_name  }} to ${{ inputs.environment || 'integration' }}
+
+on:
+  workflow_dispatch:
+    inputs:
+      gitRef:
+        description: 'Commit, tag or branch name to deploy'
+        required: true
+        type: string
+      environment:
+        description: 'Environment to deploy to'
+        required: true
+        type: choice
+        options:
+        - integration
+        default: 'integration'
+      ecrRepositoryName:
+        description: 'ECR repo name to push image to'
+        required: true
+        type: choice
+        options:
+          - publisher-on-postgres-branch
+        default: 'publisher-on-postgres-branch'
+  release:
+    types: [released]
+
+jobs:
+  build-and-publish-image:
+    if: github.event_name == 'workflow_dispatch' || startsWith(github.event.release.tag_name, 'v')
+    name: Build and publish image
+    uses: alphagov/govuk-infrastructure/.github/workflows/build-and-push-multiarch-image.yml@main
+    with:
+      gitRef: ${{ inputs.gitRef || github.event.release.tag_name }}
+      ecrRepositoryName: ${{ inputs.ecrRepositoryName }}
```

```suggestion
      ecrRepositoryName: publisher-on-postgres-branch
```
...and hardcode this instead.

---

makes sense